### DISCUSSION TITLE
ESLint config

### DISCUSSION THREAD

**Discussion by Kamahl19** [vitest-dev/vitest#1571] [Q&A]
Hey, whats the best practice for eslint config?

Should we set `env: { 'jest/globals': true }` ? And what about the `eslint-plugin-jest`? Is that reusable?

**Comment by kingyue737** [comment]
I add the following rule as Typescript has already checked type. No need to let ESlint check again.
```js
'no-undef': 'off'
```

**Comment by nickserv** [comment]
That depends more on the tools you're using. For example for Vue you should use `create-vue`, and for React I would recommend `eslint-config-react-app` (from `create-react-app`) until something better comes along.

**Reply by farazshuja** [reply]
You can try https://github.com/saqqdy/eslint-plugin-vitest-globals
